src/modules/auth/auth.module.ts

```diff
@@ -3,8 +3,6 @@ import { AuthService } from './auth.service';
 import { AuthController } from './auth.controller';
 import { JwtModule } from '@nestjs/jwt';
 import { UsersModule } from '../users/users.module';
-import { UsersRepository } from '../users/users.repository';
-import { DataSource } from 'typeorm';
 
 @Module({
   imports: [
```

Comment:
I think we do not have AUTH_TOKEN_SECRET or AUTH_TOKEN_EXPIRES_IN env variables. You should change those names to AUTH_ACCESS_TOKEN_SECRET or AUTH_ACCESS_TOKEN_EXPIRES_IN